app/src/main/java/com/tft_mvvm/app/ui/adapter/ViewPagerAdapter.kt

```diff
@@ -0,0 +1,24 @@
+package com.tft_mvvm.app.ui.adapter
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentStatePagerAdapter
+import java.util.*
+
+class ViewPagerAdapter(
+    fragmentManager: FragmentManager,
+    private val titleFragment: ArrayList<String>,
+    private val fragmentList: ArrayList<Fragment>
+) : FragmentStatePagerAdapter(fragmentManager) {
+
+    override fun getItem(p0: Int): Fragment = fragmentList[p0]
+
+    override fun getCount(): Int = fragmentList.size
+
+    override fun getPageTitle(p0:Int):CharSequence = titleFragment[p0]
```

Comment:
Your code isn't formatted here and params should be renamed.
`p0 `→ `position `